rust/actyx/ax-core/src/api/events/service.rs

```diff
@@ -607,7 +607,10 @@ async fn store_line(store: &BanyanStore, line: &str) -> anyhow::Result<()> {
     let line: Line = serde_json::from_str(line)?;
     let timestamp = line
         .timestamp
-        .or_else(|| line.time.and_then(|t| t.parse().ok()))
+        .or_else(|| {
+            line.time
+                .and_then(|t| t.parse::<AqlTimestamp>().map(Timestamp::from).ok())
```

Comment:
we could also add `Timestamp::parse()` method (no trait); this would be a great place where to document why there is no `impl FromStr for Timestamp`